.github/workflows/e2e-component-tests-embedding-sdk.yml

```diff
@@ -147,19 +194,16 @@ jobs:
         with:
           m2-cache-key: "cljs"
 
-      - name: Compile CLJS
-        run: yarn build-pure:cljs
-        shell: bash
-
-      - name: Build Embedding SDK package
-        run: yarn build-embedding-sdk
-
       - name: Run Metabase
         run: node e2e/runner/run_cypress_ci.js start
 
       - name: Make app db snapshot
         run: node e2e/runner/run_cypress_ci.js snapshot --browser ${{ steps.cypress-prep.outputs.chrome-path }}
 
+      # Has to be run after the app db snapshot is made, as it is not compatible with React 19
+      - name: Change React version to ${{ matrix.react-version }}
+        run: ./bin/embedding-sdk/change-react-version.bash ${{ matrix.react-version }}
```

Comment:
This can be ran locally too. Let me know if you'd like me to modify `run_cypress_local.ts` with a variable like `REACT_VERSION` that essentially do `change-react-version 19 && do_stuff && change-react-version restore` for you.